docs/resources/aws_db_subnet_group.md

```diff
@@ -0,0 +1,71 @@
+---
+title: About the aws_db_subnet_group Resource
+platform: aws
+---
+
+# aws\_db\_subnet\_group
+
+Use the `aws_db_subnet_group` InSpec audit resource to test properties of a db subnet group.
+
+## Syntax
+
+ An `aws_db_subnet_group` resource block uses the parameter to select a subnet group.
+
+    describe aws_db_subnet_group(db_subnet_group_name: 'subnet-group-name-12345') do
+      it { should exist }
+    end
+
+
+#### Parameters
+
+##### db\_subnet\_group\_name _(required)_
+
+This resource accepts a single parameter, the DB Subnet Group Name.
+This can be passed either as a string or as a `aws_db_subnet_group: 'value'` key-value entry in a hash.
+
+See also the [AWS documentation on DB Subnet Groups](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_VPC.WorkingWithRDSInstanceinaVPC.html#USER_VPC.Subnets).
+
+
+## Properties
+
+|Property                     | Description|
+| ---                         | --- |
+|db\_subnet\_group\_name    | The name of the DB subnet group. |
+|db\_subnet\_group\_description    | Provides the description of the DB subnet group. |
+|vpc\_id    | Provides the VpcId of the DB subnet group. |
```

Comment:
Perhaps VPC ID or VPC Id could read better than VpcId.

Reply:
Updated in recent commit